### ISSUE TITLE
Jumping sticky header on OS X

### ISSUE THREAD

**Issue by Maltson** [Mottie/tablesorter#1442]
On OS X in chrome there is a problem with a sticky header. If you set the parameter stickyHeaders_attach, then when you scroll the header starts to jump. This can be seen even in the example with the name "Attach sticky header to its parent" https://mottie.github.io/tablesorter/docs/example-widget-sticky-header.html

**Comment by Mottie** [comment]
Hi @Maltson!

I'm not sure what to tell you, or to suggest. I am only seeing this same behavior in MS Edge, but not in any other of the browsers I tested - I don't own any Apple products, so I can't test this in Safari.

I see the same behavior for the [css sticky headers widget](https://mottie.github.io/tablesorter/docs/example-widget-css-sticky-header.html) but I'm not sure how to fix the problem in either case.

Maybe once [`position: sticky`](http://caniuse.com/#search=sticky) is supported in Edge, I can test and add another option to use sticky instead of a fixed position, but there are no guarantees that it won't misbehave either.

**Comment by Maltson** [comment]
Hi @Mottie 
The problem in the Chrome browser. I recorded a video with a problem:
https://monosnap.com/file/fXmFjkrdscom8ZqTRfIRMXnzUkjC06

**Comment by Mottie** [comment]
Yes, Chrome on OSX... it's smooth as butter on my Windows system. As I said, I don't own any i-devices, so I can't troubleshoot this problem. If you find a solution, I would be happy to include the fix.

**Comment by Mottie** [comment]
This *should* be fixed in v2.29.0. Please let me know.

**Comment by alex-rxflo** [comment]
This is still going on in chrome with tablesorter version 2.29.0.  It is occurring in your example that @Maltson took a video of, and also in my implementation.

**Comment by alex-rxflo** [comment]
For additional context, here are the specs of my graphics card and displays.

<img width="426" alt="screen shot 2017-12-11 at 10 50 50 am" src="https://user-images.githubusercontent.com/33532188/33845682-795d2828-de61-11e7-9fca-a1a5ec257dcf.png">

I tested this on my external (benq) monitors, and the issue seems to go away. It must be an issue specific to the retina display or something.
